packages/expo-haptics/src/Haptics.types.ts

```diff
@@ -36,4 +36,12 @@ export enum ImpactFeedbackStyle {
    * A collision between large, heavy user interface elements.
    */
   Heavy = 'heavy',
+  /**
+   * A collision between user interface elements that are soft, exhibiting a large amount of compression or elasticity.
```

Comment:
```suggestion
   * A collision between user interface elements that are soft and show a large amount of compression or elasticity.
```

We should avoid using `,` for adding a pause to the sentence. It makes it unclear and a bit harder to understand. Either break it into two sentences or use a conjunction like the above suggestion.

Reply:
This doc was copied straight from the official Apple docs. Also, the usage of comma here implies the "large amount of elasticity" is describing the impact itself, whereas your suggestion makes it aound like "the component shows a large amount of elasticity".

Reply:
Got it. Yeah, the changes I suggested do alter the context. I didn't have context and thought it was related to the impact of UI elements. I don't think it is trivial when it comes to communicating in a clear way as to what something does. However, the context you've provided now, it helps me understanding this and also recognize the same pattern as you mentioned in enum definitions.

Reply:
Thanks :D